app/assets/javascripts/components/physical_infrastructures/physical-rack-toolbar.js

```diff
@@ -0,0 +1,50 @@
+ManageIQ.angular.app.component('physicalRackToolbar', {
```

Comment:
This is an older approach, this file should not be needed at all anymore, this will be handled by common code introduced in https://github.com/ManageIQ/manageiq-ui-classic/pull/3436.

Reply:
Cool @himdel :+1: for the new approach

Reply:
(The new approach should be ready to use, that PR is merged.)

Reply:
Done :smile: